sql/hive/src/main/scala/org/apache/spark/sql/hive/MetastoreRelation.scala

```diff
@@ -142,6 +140,27 @@ private[hive] case class MetastoreRelation(
     }
   )
 
+  private def getSize(f: Path): Long = {
```

Comment:
Renamed the method, added documentation and also returning the default bytes when size > broadcast threshold.